modules/ROOT/pages/6.0.3-release-notes.adoc

```diff
@@ -0,0 +1,94 @@
+
+With this update, selecting multiple cells, invoking the cell properties dialog, and removing `height` values across the selected cells works as expected, with all cell heights deleted at once.
+
+=== When multiple rows were selected, removing values in the row properties dialog did not work
+
+Previously, when dealing with {productname} tables with two or more rows, the row properties dialog box did not honour deleting values when multiple rows were selected.
+
+If multiple table rows were selected in a {productname} table with two or more columns and the cell properties dialog was invoked, deleting the height value in the dialog box had no effect. However, if each cell was selected individually, invoking the cell properties dialog and deleting the height value worked as expected.
+
+With this update, selecting multiple cells, invoking the cell properties dialog, and removing height values across the selected cells works as expected, with all cell heights deleted at once.
+
+NOTE: This is related to the near-equivalent bug regarding cell properties deletion not being honored when multiple cells were selected. The two, slightly different, behaviors are being documented separately for clarity’s sake, and because some people only encountered one or other errant behavior.
+
+=== `format_empty_lines` did not work as documented
+
+Previously, when `format_empty_lines` was set to `true`, lines that included no text (ie, a line consisting of just a Return) did not preserve inline formatting when saved.
+
+So, for example, a line
+
+[source,html]
+----
+<p> 
+<em> 
+<strong> 
+<span style="font-family: arial, helvetica, sans-serif;" ><br /></span> 
+</strong> 
+</em> 
+</p> 
+----
+
+when saved, became
+
+[source,html]
+----
+<p> 
+&nbsp; 
+</p> 
+----
+
+That is, the line was preserved, but the formatting was not.
+
+With this update, `format_empty_lines: true` behaves as it should (and as advertised), and formatting added to an otherwise empty line is preserved when saved.
+
+NOTE: by default `format_empty_lines` is set to `false`.
+
+
+=== Some inline elements specified by the {productname} schema were not removed by default when empty
+
+By default, the {productname} schema removes empty inline +html+ elements.
+
+Previously, however, some inline +html+ elements — including +s+, +u+, +var+, +cite+, +dfn+, +code+, +mark+, +q+, +sup+, +sub+, and +samp+ — were not removed when they were present but empty.
+
+With this update, inline +html+ elements such as those noted above will have the +removeEmpty+ schema flag set, as expected.
+
+=== The default {productname} schema did not include the `<s>` element
+
+Previously, the default {productname} schema did not include the `+<s>+` element. Consequently, formatting text with `Strike-through` did not render as expected.
```

Comment:
```suggestion
Previously, the default {productname} schema did not include the `+<s>+` element in the text inline elements section of the schema. The element was included in the full element list, however.

Consequently, although `Strike-through` elements rendered as expected, anything that relied on the `schema.getTextInlineElements()` API may not have handled the `+<s>+` element correctly.

This update corrects the omission in the {productname} schema and `+<s>+` elements are now handled correctly by the `schema.getTextInlineElements()` API.
```

A re-write correcting the above.

Reply:
Should `Strike` be capitalised? Also re the last sentence, it's not really "handled" by that API it's just included in the returned list. So maybe something like this?
```
This update corrects the omission in the {productname} schema and `+<s>+` element is now correctly returned by the `schema.getTextInlineElements()` API.
```

Reply:
Edited my suggestion as per your note. Thanks.